src/ssh/sshConfig.ts

```diff
@@ -73,6 +74,84 @@ export default class SSHConfiguration {
         return new SSHConfiguration(config);
     }
 
+    static async loadGitpodSSHConfig(): Promise<SSHConfiguration> {
```

Comment:
nothing to change just to my curiosity what is advantage/disadvantage of using vscode.fs api instead? VS Code should be better aware of changes then?

Reply:
not sure, maybe it has some advantage when reading/writing on workspace files but for files outside workspace I image is the same, also it will work for other special vscode schemas like remote but we don't need it here